runtime/compiler/env/CHTable.cpp

```diff
@@ -249,6 +253,13 @@ void TR_CHTable::cleanupNewlyExtendedInfo(TR::Compilation *comp)
 //
 bool TR_CHTable::commit(TR::Compilation *comp)
    {
+#if defined(JITSERVER_SUPPORT)
+   if (TR::CompilationInfo::getStream())
```

Comment:
Since you have comp object, it's probably better to test comp->isOutOfProcessCompilation(). This way maybe you don't need to include `CompilationRuntime.hpp`